sentences/de/_common.yaml

```diff
@@ -48,7 +48,7 @@ lists:
         out: "Fahrenheit"
 expansion_rules:
   name: "[(der | den | die | das)] {name}"
-  area: "(in (der | dem) | im) {area}"
+  area: "[(in (der | dem) | im | der | die | das)] {area}"
```

Comment:
tbh, I don't get it, what this brings us 🙈
do you have an example, what is now covered?

Reply:
hi @Scorpoon, could you please answer @diplix's question?